index.html

```diff
@@ -18,20 +18,33 @@
     </div>
     <div class="main">
         <div class="question__photo">
-            <img class="question__image" src="img/QuestionPhoto.png" alt="" >
+            <img class="question__image" src="img/QuestionPhoto_Vader.png" alt="" >
         </div>
         <div class="mode">
-            <p class="mode__type">MODE: Who is this character?</p>
-            <div class="mode__rules">
+            <p class="mode__type">Question: Who is this character?</p> 
+            <!-- <div class="mode__rules">
```

Comment:
chyba nie rozumiem czemu to jest zakomentowane? chyba to powinno być na wierzchu a zakomentowane Twoje zmiany? (albo np dodane na razie display none?)